Source/AliveLibAE/MainMenu.cpp

```diff
@@ -2696,7 +2696,13 @@ MainMenuNextCam MainMenuController::ControllerMenu_Update_4D16D0(u32 input)
         return MainMenuNextCam(MainMenuCams::eOptionsCam);
     }
 
-    if (input & InputCommands::Enum::eConfigure)
+#if ORIGINAL_PS1_BEHAVIOR // OG Change - Allows player to select "Configure Controller" using their gamepad (Square/Circle Button)
```

Comment:
shouldn't this be AUTO_SWITCH_CONTROLLER as well?

Reply:
It's unrelated. I'll create a new cmake option for this one too.